pkg/apis/core/types_seed.go

```diff
@@ -104,6 +104,8 @@ type SeedStatus struct {
 	// Allocatable represents the resources of a seed that are available for scheduling.
 	// Defaults to Capacity.
 	Allocatable corev1.ResourceList
+	// ClientCertificateExpirationTimestamp is the Kubernetes version of the seed cluster.
+	ClientCertificateExpirationTimestamp *metav1.Time
```

Comment:
Nit: The name of the field is a bit long. While this is not a big problem of course, perhaps it could be shortened to `ClientCertificateExpiration` or `ClientCertificateExpiresOn`? I think both these names already imply a `Time` type, so `Timestamp` adds to the length without adding much additional information. Of course if we rename it here, it should be renamed everywhere else for consistency.

Reply:
I added the `Timestamp` suffix because of similarity to `.metadata.creationTimestamp` or `.metadata.deletionTimestamp`. It's a bit long, I agree, but for similarity reasons I'd keep it for now unless you insist on changing it.